src/main/java/mil/dds/anet/beans/Task.java

```diff
@@ -178,6 +184,16 @@ public Organization getResponsibleOrg() {
 		return responsibleOrg.getForeignObject();
 	}
 
+	@GraphQLQuery(name="reports")
+	public CompletableFuture<AnetBeanList<Report>> loadReports(@GraphQLRootContext Map<String, Object> context, @GraphQLArgument(name="query") ReportSearchQuery query) {
+	// TODO: Use the query parameter 
+		if (reports != null) {
+			return CompletableFuture.completedFuture(reports);
+		}
+		return AnetObjectEngine.getInstance().getTaskDao().getReportsForTask(context, uuid)
+				.thenApply(o -> { reports = new AnetBeanList<Report>(o); return reports; });
```

Comment:
When only a totalCount is needed, there is probably a better way to get this from the ResultSet. Could be good to make `AnetBeanList.getList` lazy, and be the one that triggers parsing of the Report beans